### DISCUSSION TITLE
Welcome to codedoc Discussions!

### DISCUSSION THREAD

**Discussion by loreanvictor** [CONNECT-platform/codedoc#92] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TysonMN** [comment]
Hello everyone :)

My name is Tyson Williams.  I joined this community as part of starting [my blog](https://tysonwilliams.coding.blog/).  I collected blog post ideas for about a year before `coding.blog` and the CONNECT platform started.  I was waiting for a blogging platform that made it easy to write about code, and this is definitely it!  I love that everything is open source because I enjoy improving small details that would be impossible to convince someone else to do.  It is easy to convince someone to accept an improvement if all they have to do merge in a PR.  Best of all is @loreanvictor.  He patient, considerate, and quickly addresses new issues.

I expect my involvement with the CONNECT platform to increase in the future.  In particular, I expect to create my first open source project in the form of a [Google Analytics plugin](https://github.com/TysonMN/codedoc-google-analytics).

Here's to a great 2021 for `coding.blog` and the CONNECT platform 🥂
